msal_extensions/__init__.py

```diff
@@ -1 +1,3 @@
 __version__ = "0.0.1"
+
+from ._cache_lock import CrossPlatLock
```

Comment:
Totally fair! Do you have a preference on whether I fix this by hiding it further or making it more public?

Just out of curiosity, could you explain why that's more Pythonic? In general, I've struggled to get my head around the Python idioms around organizing code. My line of thinking for putting it this way is that it forces folks to import it in a single consistent way: `from msal_extensions import CrossPlatLock`. That way if we want to rename the file in the future it's not a breaking change.

Reply:
I guess, by my own reasoning, my preferred option would be to hide it further to prevent avenues for breaking changes in the future. If folks demand access to this helper, we can always make it public later.

Reply:
Fixed in bda687b